pkg/builder/quarkus.go

```diff
@@ -143,14 +143,16 @@ func GenerateQuarkusProjectCommon(camelQuarkusVersion string, runtimeVersion str
 					},
 				},
 			},
+			Extensions: true,
 		},
 	)
 
 	return p
 }
 
 func buildQuarkusRunner(ctx *builderContext) error {
-	mc := maven.NewContext(path.Join(ctx.Path, "maven"))
+	// Build the project using the new Maven daemon build strategy 
+	mc := maven.NewContext(path.Join(ctx.Path, "mvnd"))
```

Comment:
That argument is the directory where the Maven project is created, that can be kept as is.